feature-libs/product-configurator/rulebased/components/overview-filter/configurator-overview-filter.component.html

```diff
@@ -0,0 +1,78 @@
+<ng-container *ngIf="config$ | async as config">
+  <ng-container *ngIf="showFilterBar">
+    <cx-configurator-overview-filter-bar
+      [config]="config"
+    ></cx-configurator-overview-filter-bar>
+  </ng-container>
+  <div class="cx-overview-filter-header">
```

Comment:
Guess i will create dedicated i18n props for the a11y texts instead of concatenating. Concatenating works with the terms we choose for englisch texts, but this might not be true for other languages.

Reply:
That's even better.